authority/provisioner/oidc_options.go

```diff
@@ -0,0 +1,90 @@
+package provisioner
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"text/template"
+	"time"
+
+	"github.com/coreos/go-oidc/v3/oidc"
+)
+
+type ProviderJSON struct {
+	IssuerURL   string   `json:"issuer,omitempty"`
+	AuthURL     string   `json:"authorization_endpoint,omitempty"`
+	TokenURL    string   `json:"token_endpoint,omitempty"`
+	JWKSURL     string   `json:"jwks_uri,omitempty"`
+	UserInfoURL string   `json:"userinfo_endpoint,omitempty"`
+	Algorithms  []string `json:"id_token_signing_alg_values_supported,omitempty"`
+}
```

Comment:
It would be nice if this is available in something like a `/.well-known/openid-configuration`

Reply:
Agree, but that's something Wire would have to implement/provide, in that case, I think? It could be an (optional) improvement after discussing it with them?